src/components/ContentListTable/ContentListTable.tsx

```diff
@@ -45,14 +64,20 @@ const useStyles = createUseStyles({
 const ContentListTable = () => {
   const classes = useStyles();
   const queryClient = useQueryClient();
-
   const storedPerPage = Number(localStorage.getItem('perPage')) || 20;
 
   const [page, setPage] = useState(1);
   const [perPage, setPerPage] = useState(storedPerPage);
 
   const hasActionPermissions = true; // TODO: Incorporate permissions here later.
 
+  const {
+    isLoading: repositoryParamsLoading,
+    error: repositoryParamsError,
+    isError: repositoryParamsIsError,
+    // data: repositoryParamsData,
+  } = useRepositoryParams();
```

Comment:
I added this call within the main listTable so that I can access the error/isError/isLoading states and merge them with the other loading/error states below.